astropy/wcs/wcsapi/high_level_api.py

```diff
@@ -53,8 +56,11 @@ def array_index_to_world(self, *index_arrays):
         Convert array indices to world coordinates (represented by Astropy
         objects).
 
-        See `~astropy.wcs.wcsapi.BaseLowLevelWCS.array_index_to_world_values` for pixel indexing and
-        ordering conventions.
+        If a single high-level object is used to represent the world
+        coordinates, it is returned as-is (not in a tuple/list), otherwise a
```

Comment:
(goes with the other suggestion)
```suggestion
        coordinates (i.e., ``world_n_dim`` is 1), it is returned as-is (not in a tuple/list), otherwise a
```

Reply:
As above.